daal4py/sklearn/monkeypatch/dispatcher.py

```diff
@@ -88,6 +91,8 @@
 if daal_run_version >= (2020, 1) and daal_link_version >= (2020, 1):
     _mapping['fin_check'] = [[(validation, '_assert_all_finite', _daal_assert_all_finite), None]]
 
+if daal_run_version >= (2020, 2) and daal_link_version >= (2020, 2):
```

Comment:
Will It be  this changes in oneAPI releases? Do you need to check it?

Reply:
Yes, oneDAL master has version (2021, 8). I checked that optimization is used.

Reply:
But in version oneDAL < beta08 we will have a failed. because 2021.*07 > 2020.*02

Reply:
I added more complex and working version getting. It should work for both beta and mnt.